### ISSUE TITLE
Refresh the docProperty field in Ms Word onlline document

### ISSUE THREAD

**Issue by ishanig11** [OfficeDev/office-js#4426]
I want to refresh Word.FieldType.docProperty field
field.result.insertText("test", "Replace"); this function works fine in desktop version. But its not working for online word.

Can you please tell me how to refresh these fields in Word online version.

Thanks in advance.

**Comment by yilin4** [comment]
@ishanig11 Now, the online version only has 4 field types, only these 4 field types can be updated for online word.
![image](https://github.com/OfficeDev/office-js/assets/107384367/31b31947-8cc5-46d9-b0ef-1113942a3de8)
So if your document has the docProperty field, the online Word can't recognize it. If you have this need, we would like to suggest you go to https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform to ask a new question. Feature requests on it are considered when we go through our planning process.

**Comment by ishanig11** [comment]
Is there anyway to speed up this enhancement request?

**Comment by ishanig11** [comment]
Is there anyway to expediate this product feature as its really required for our functionality development.

**Comment by yilin4** [comment]
At this time, expediting the implementation of the requested feature is not in our current plans. However, we are keen to understand more about your specific use case and the potential business impact this feature would have. Gaining a deeper insight into your needs will enable us to better align our future development efforts with your priorities.
If you would like to follow up with us and provide more details, please use this form to submit your information:
https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR9NvrlmrippKsMvh4kTM325URVZMWjYyQTVOTVNLTUo5VzFRTTcyNU1RSy4u

**Comment by ishanig11** [comment]
Thank you @yilin4 : I have added details in the above link.
How can I track that request?

**Comment by stevefengjin** [comment]
> Thank you @yilin4 : I have added details in the above link. How can I track that request?

@ishanig11 we have added this request into our internal product backlog, you can track the feature request that you submitted in our idea pool.

**Comment by ishanig11** [comment]
@stevefengjin : Can you share link for feature release plan? Which all features are planned for next release and next release date?

**Comment by stevefengjin** [comment]
@ishanig11 Thank you for your inquiry regarding our feature release plan. While we appreciate your interest, we are unable to share specific details about upcoming features and release dates at this time. Rest assured, our team is diligently working on enhancements to improve our platform. We recommend keeping an eye on official Office.JS Add-ins documentation site or M365 DevBlogs for updates and announcements, there is also a monthly community call where we proactively share the updates and developments of our platform.  If you have any specific requirements or suggestions, please feel free to share them with us via this form: https://forms.office.com/r/QayRq1FVEb.

**Comment by ishanig11** [comment]
@stevefengjin : Yes, I have already logged request using office form. But I could not find anyway to track my request.

**Comment by shanshanzheng-dev** [comment]
Hi @ishanig11 Thanks for logging your request. If we end up working on this issue, we will report back here and update our process. We value your feedback.

**Comment by aarzoomittal-microsoft** [comment]
Thank you for the discussion and for sharing the use case.
 
After reviewing this based on the current Office.js API design and the points raised in the comments, the behavior described is expected and aligns with the existing API surface. Supporting this scenario would require new or expanded API functionality rather than a fix to current behavior.
 
As this is a feature request rather than a product bug, we track such requests through the Microsoft 365 Developer Platform feedback channels (we have one for this - [https://techcommunity.microsoft.com/idea/microsoft365developerplatform/refresh-the-docproperty-field-in-ms-word-online-document/4173375](url)). We’ll close this issue here to keep the bug tracker focused; we appreciate the feedback and discussion.
